Smartling.API/Exceptions/ThrottlingException.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace Smartling.Api.Exceptions
+{
+  public class ThrottlingException : Exception
```

Comment:
Maybe create the base exception like `SmartlingApiException` since we touched this. It will allow to distinct smartling exceptions from all other exceptions. We don't need this in integration so far, but it may allow generalizing some error handling in the future.

Reply:
@dimitrystd Done.